### ISSUE TITLE
Find out if it's possible to toggle the screen reader on/off on a per-interview basis

### ISSUE THREAD

**Issue by nonprofittechy** [SuffolkLITLab/docassemble-AssemblyLine-documentation#192]
T&P 22 - suggested that screen reader should be made optional or togglable. This sounds confusing to me (QS) but it at least should be possible to turn it off on a per-interview or per-organization basis.

Note that we added the screen reader NOT for blind users, but for people with low literacy. It's also a feature people might need on shared workstations or when borrowing a mobile phone.

We should consider:

* Doing some analytics on usage, if we can figure out how.
* Figuring out ways to make it less prominent, but still discoverable.

**Comment by plocket** [comment]
I think this might be the GitHub documentation: https://docassemble.org/docs/special.html#speak_text

**Comment by plocket** [comment]
When added to the docs, should this be in a section about the interview order block or some other block, or should this be in the section about writing good interviews?

**Comment by nonprofittechy** [comment]
Magic variable documentation
